### DISCUSSION TITLE
DDI with large molecule model

### DISCUSSION THREAD

**Discussion by teutonicod** [Open-Systems-Pharmacology/Forum#1097] [Questions & Problems] [answered]
Dear all,

I build a model containing small and large molecule together, since the molecules are connected. I would like to make a DDI prediction, for instance with Itraconazole, on the small molecules that I have in my model, but as you all know, all the model we have for DDI are built on the small molecule model. 
Do you have any idea (any trick!) to combine a small and large model together? For instance, in MoBi? 
Or for instance, do you think that I could import itraconazole in my simulation with the large molecule model and then reparametrize the model to make it identical to the original one build in the small molecule model?

FYI, I already checked and changing the model from small to large on the itraconazole-midazolam interaction has a big impact, and the predictions are not accurate anymore.

Please any input is welcome!

Thanks in advance!

Donato

**Comment by Christoph27** [comment, marked as answer]
Hi Donato!

If you use a small molecule within the large molecule model, the only difference compared to using the small molecule model is, that you introduce an (effective) barrier between the plasma and interstitial space of the organs. If the fu is small (which is the case for e.g., Itraconazole, this matters). All you have to do is to adjust the permeability P_endothelial for your drug (and the metabolites) for all organs except brain to the value which is used for the small molecule model (100 cm/min, i.e. fast exchange) and you will end up with exactly the same model as if you had chosen the small molecule model! You can stay in PK-Sim for this. Just take care that you also change the P_endothelial (= P(plasma<->interstitial)) values for the mucosa “organs” (the latter are not accessible in the Permeability/P_endothelial group but in the Neighborhoods in the Hierarchy view). 

So just using the large molecule model for all compounds (large and small molecules) and make the adjustments described above in case the PBPK model for the small molecule  was not established using the large molecule model, should do the trick. 

Best, Christoph

**Reply by teutonicod** [reply]
Hi @Christoph27,

Thanks a lot for the very clear explanation! I followed your steps and I was able to recreate the interaction Itraconazole-Midazolam in the large molecule model with basically identical PK and (almost) identical AUCR. Thanks a lot, your suggestion was exactly what I needed!

Donato
